for ... else ... : 
2.without break

函数注解 : def add(num1:int,num2:int)->int:

with : 
enter!
exc_type : <class 'TypeError'>
exc_value : must be str, not int
traceback : <traceback object at 0x7f7b67120788>
exit!

yield : 
<generator object yield_test at 0x7f7b671c54c0>
0
1
2
3
4
5
6
7
8
9
decorator : 
Hello World!
l am a decorator!
yield and send
please tell me what you want to say
l receive your words : hello
with no super : 
Hello,everyone!
l am his friend

with super : 
Hello,everyone!
l am his friend

property : 
170 50
property_test(55,50)
property_test(55,175)
Help on class property_test in module __main__:

class property_test(builtins.object)
 |  Methods defined here:
 |  
 |  __init__(self, height, weight)
 |      Initialize self.  See help(type(self)) for accurate signature.
 |  
 |  __repr__(self)
 |      Return repr(self).
 |  
 |  ----------------------------------------------------------------------
 |  Data descriptors defined here:
 |  
 |  __dict__
 |      dictionary for instance variables (if defined)
 |  
 |  __weakref__
 |      list of weak references to the object (if defined)
 |  
 |  height
 |  
 |  weight

None

kwargs and args
nick
('foo', 'bar')
{'sex': 'male'}
[0, 1, 4, 9, 16, 25, 36, 49, 64, 81]
{0: 0, 1: 1, 2: 4, 3: 9, 4: 16, 5: 25, 6: 36, 7: 49, 8: 64, 9: 81}
lambda : 
5
classmethod : 
l am classmethod
cls.name : nick
l am called with classmethod cls
1 2 [3, 4]
1 [2, 3] 4
num_use_ :　
1000000500
10
slots in class : 
you can't add attribute that is not in slots
hello, nick
your name is too short!
1
2
<class 'ellipsis'>
Traceback (most recent call last):
  File "/tmp/832175568/main.py", line 221, in <module>
    print(1 / 0)
ZeroDivisionError: division by zero

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/tmp/832175568/main.py", line 224, in <module>
    raise ExitBye("have an error with raise") from e
ExitBye: have an error with raise

